Framework/Geometry/inc/MantidGeometry/Objects/CSGObject.h

```diff
@@ -259,14 +266,14 @@ class MANTID_GEOMETRY_DLL Object {
   std::unique_ptr<Kernel::Material> m_material;
 
 protected:
-  std::vector<const Surface *>
-      SurList; ///< Full surfaces (make a map including complementary object ?)
+  std::vector<const Surface *> m_SurList; ///< Full surfaces (make a map
+  /// including complementary object ?)
 };
 
 /// Typdef for a shared pointer
-typedef boost::shared_ptr<Object> Object_sptr;
+typedef boost::shared_ptr<CSGObject> Object_sptr;
```

Comment:
Shouldn't the typedefs be renamed to CSGObject_sptr?

Reply:
I'm hoping there are none remaining.